help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -102,48 +113,51 @@ Read [Activate profiles and segments to streaming segment export destinations](/
+To map your XDM fields to the [!DNL Oracle Eloqua] destination fields, follow these steps:
 
 1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
 1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
-1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
-    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Oracle Eloqua] instance:
-        |Source Field|Target Field| Mandatory|
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and from the list of [[!DNL update a contact API endpoint fields]](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html) pick the desired attribute name and type in the **[!UICONTROL Attribute name]** field.
+    * Repeat these steps to add both the required and any desired attribute mappings between your XDM profile schema and [!DNL Oracle Eloqua]:
+        | Source Field | Target Field | Mandatory |
         |---|---|---|
-        |`xdm: personalEmail.address`|`Attribute: emailAddress`| Yes |
         |`IdentityMap: Eid`|`Identity: EloquaId`| Yes |
-
-    * An example using these mappings is shown below:
+        |`xdm: personalEmail.address`|`Attribute: emailAddress`| Yes |
+        |`xdm: personName.firstName`|`Attribute: firstName`| |
+        |`xdm: personName.lastName`|`Attribute: lastName`| |
+        |`xdm: workAddress.street1`|`Attribute: address1`| |
+        |`xdm: workAddress.street2`|`Attribute: address2`| |
+        |`xdm: workAddress.street3`|`Attribute: address3`| |
+        |`xdm: workAddress.postalCode`|`Attribute: postalCode`| |
+        |`xdm: workAddress.country`|`Attribute: country`| |
+        |`xdm: workAddress.city`|`Attribute: city`| |
+
+    * An example with the above mappings is shown below:
     ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/email-marketing/oracle-eloqua-api/mappings.png)
 
-        >[!IMPORTANT]
-        >
-        >Both the `emailAddress` and `EloquaId` target attribute mappings are mandatory.
+>[!IMPORTANT]
+>
+>* Attributes specified in the **[!UICONTROL Target field]** should be named exactly as specified in the [[!DNL update a contact API endpoint fields]](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html) as these attributes will form request body.
```

Comment:
Line 142: Let's link to the Create a contact (https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-post.html) page instead of update contact.

Reply:
This isn't updated yet. Please correct both the link and the link text.